webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
```

Comment:
I'm happy to implement it too. :) if you tell me what I should do. As you prefer.